src/components/Account/Login/Login.actions.test.js

```diff
@@ -86,4 +105,237 @@ describe('actions', () => {
+    expect(changeVoiceAction.lang.substring(0, 2)).toBe(
+      initialState.language.lang.substring(0, 2)
+    );
+  });
+  it('should log in and change voice to voice stored on DB even if the voice dialect is not equal to the app dialect', async () => {
+    const initialSpeech = {
+      voices: voicesMock,
+      options: {
+        voiceURI: 'Español-Colombiano-mock',
+        lang: 'es-ES'
+      }
+    };
+    initialState = {
+      ...initialState,
+      speech: initialSpeech,
+      language: { lang: 'en-US' }
+    };
+    const storeWithSpeech = mockStore(initialState);
+    const user = { email: 'test', password: '1122' };
+    await storeWithSpeech
+      .dispatch(actions.login(user, 'local'))
+      .catch(() => {});
+
+    const dispatchedActions = storeWithSpeech.getActions();
+    const changeVoiceAction = dispatchedActions[dispatchedActions.length - 1];
+    expect(changeVoiceAction.type).toBe(CHANGE_VOICE);
+    expect(changeVoiceAction.lang.substring(0, 2)).toBe(
+      initialState.language.lang.substring(0, 2)
+    );
+    expect(changeVoiceAction.lang).not.toBe(initialState.language.lang);
+  });
+  it('should set empty voice if voices are unavailable for the app language', async () => {
+    const initialSpeech = {
+      voices: [
+        {
+          lang: 'es-ES',
+          voiceURI: 'Español-Colombiano-mock',
+          voiceSource: 'local',
+          name: 'Español-Colombiano-mock'
+        }
+      ],
+      options: {
+        voiceURI: 'Español-Colombiano-mock',
+        lang: 'es-ES'
+      }
+    };
+    initialState = {
+      ...initialState,
+      speech: initialSpeech,
+      language: { lang: 'en-GB' }
+    };
+    const storeWithSpeech = mockStore(initialState);
+    const user = { email: 'test', password: '1122' };
+    await storeWithSpeech
+      .dispatch(actions.login(user, 'local'))
+      .catch(() => {});
+
+    const dispatchedActions = storeWithSpeech.getActions();
+    const changeVoiceAction = dispatchedActions[dispatchedActions.length - 1];
+    expect(changeVoiceAction.type).toBe(CHANGE_VOICE);
+    expect(changeVoiceAction.lang.substring(0, 2)).not.toBe(
+      initialState.language.lang.substring(0, 2)
+    );
+  });
+  it('if speech options lang is null set the initial state', async () => {
+    const initialSpeech = {
+      voices: [],
+      options: {
+        voiceURI: 'empty voices',
+        lang: null
+      }
+    };
+    initialState = {
+      ...initialState,
+      speech: initialSpeech,
+      language: { lang: null }
+    };
+    const storeWithSpeech = mockStore(initialState);
+    const user = { email: 'test', password: '1122' };
+    await storeWithSpeech
+      .dispatch(actions.login(user, 'local'))
+      .catch(() => {});
+
+    const dispatchedActions = storeWithSpeech.getActions();
+    const changeVoiceAction = dispatchedActions[dispatchedActions.length - 1];
+    expect(changeVoiceAction.type).toBe(CHANGE_VOICE);
+    expect(changeVoiceAction.lang).toBe('');
```

Comment:
recommend to add a comparasion between the changeVoiceAction.voiceUri and a string 'empty voices'